config/environments/production.rb

```diff
@@ -75,7 +80,7 @@
   redis_url = ENV["REDISCLOUD_URL"]
   redis_url ||= ENV["REDIS_URL"]
   default_expiration = 24.hours.to_i
-  config.cache_store = :redis_cache_store, { url: redis_url, expires_in: default_expiration }
+  config.cache_store = :redis_cache_store, { namespace: ENV["REDIS_NAMESPACE"], url: redis_url, expires_in: default_expiration }
```

Comment:
For testing purposes, I think we need to have this also set in our development environment for our Redis setup to keep it as consistent as possible.